src/engine/surface_collision.c

```diff
@@ -450,6 +450,12 @@ static struct Surface *find_floor_from_list(struct SurfaceNode *surfaceNode, s32
     return floor;
 }
 
+// Generic triangle bounds func
+ALWAYS_INLINE static s32 check_within_bounds_y_norm(s32 x, s32 z, struct Surface *surf) {
```

Comment:
This should not be inlined

Reply:
It's hard to say if it'll be better or worse inlined, so it can be left as-is and reviewed down the line.